package.json

```diff
@@ -23,8 +28,18 @@
     "commander": "^5.0.0",
     "express": "^4.17.1"
   },
+  "peerDependencies": {
+    "next": "9 - 10"
```

Comment:
@jescalan please correct me if this line is wrong. 
We use `next` in the project as peer dependency but not declared explicitly.

We need to identify the breaking change I discussed in the PR description to agree which version we want to support

Reply:
I have read with more attention https://github.com/hashicorp/next-remote-watch/pull/20#discussion_r637987575 and yeah it probably make sense just to stick with a major release for each API change.